src/panels/config/zha/zha-add-group-page.ts

```diff
@@ -0,0 +1,202 @@
+import {
+  property,
+  LitElement,
+  html,
+  customElement,
+  css,
+  CSSResult,
+  PropertyValues,
+} from "lit-element";
+
+import "../../../layouts/hass-subpage";
+import "../../../layouts/hass-error-screen";
+import "../ha-config-section";
+import { HomeAssistant } from "../../../types";
+import {
+  ZHADevice,
+  fetchGroupableDevices,
+  addGroup,
+  ZHAGroup,
+} from "../../../data/zha";
+import "./zha-devices-data-table";
+import { SelectionChangedEvent } from "../../../components/data-table/ha-data-table";
+import { navigate } from "../../../common/navigate";
+import { PolymerChangedEvent } from "../../../polymer-types";
+import "@polymer/paper-spinner/paper-spinner";
+import "@material/mwc-button";
+import { PaperInputElement } from "@polymer/paper-input/paper-input";
+
+@customElement("zha-add-group-page")
+export class ZHAAddGroupPage extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() public narrow!: boolean;
+  @property() public devices: ZHADevice[] = [];
+  @property() private _canAdd: boolean = false;
+  @property() private _processingAdd: boolean = false;
+  @property() private _groupName: string = "";
```

Comment:
No need for this property, just read the value of the `input` when adding the group.

Reply:
So, I switched this locally to use the `@query` decorator as discussed. Unless there is something else I am missing if I do this I'll lose the ability to disable the create button when there is no name. I'd still need the change listener to check the input for a value at that point. I think the property binding is cleaner... thoughts?

Reply:
In that case, remove `_canAdd` (as it is only used to check if the name is set?) and replace that check with `_groupName`.

Reply:
(Notice that is your current code it won't disable again if you clear the name input)

Reply:
good catch. Fixed.